plugins/gopass.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: gopass
+spec:
+  version: "v0.0.3"
```

Comment:
There is a version mismatch between this manifest and what is reported by `kubectl gopass version` (see https://github.com/gopasspw/kubectl-gopass/blob/59315e46389540624cca8c6ecb45694ed1dc3e87/kubectl-gopass#L140). Have you considered to generate deployment artifacts with a script?

Reply:
Fixed, I added a small helper to create the tags and extracted the version as variable in the script.